index.html

```diff
@@ -94,103 +99,107 @@ <h2>Multi-Post Stories</h2>
+                        <li>Uber<i class="fa fa-circle"></i></li>
+                        <li>Lead Developer<i class="fa fa-circle"></i></li>
+                        <li>2023</li>
                     </ul>
                     <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                     <ul class="programming-lang">
                         <li><a href="#">HTML</a></li>
+                        <li><a href="#">Ruby on rails</a></li>
                         <li><a href="#">CSS</a></li>
                         <li><a href="#">Javascript</a></li>
                     </ul>
                     <button class="btn-action">See project</button>
                 </div>
             </div>
         </div>
+
         <div class="about-myself">
-          <div class="about-container">
-            <h2 class="hx">About Myself</h>
+            <div class="about-container">
+                <h2>About Myself</h2>
                 <p>Hello I'm a software developer! I can
-                    help you build a produt, feature or 
-                    website. Look through some of my work 
+                    help you build a produt, feature or
+                    website. Look through some of my work
                     and experience! If you like what you see
-                    and have a project you need coded, 
-                    don't hesitate to contact me. 
+                    and have a project you need coded,
+                    don't hesitate to contact me.
                 </p>
                 <a href="#" class="connect">
-                LET'S CONNECT
-            </a> 
-            <ul class="social-media">
-                <li><a href=""><img src="images/portfolio1.svg" alt=""></a></li>
-                <li><a href=""><img src="images/Linkedin icon.svg" alt=""></a></li>
-                <li><a href=""><img src="images/portfolio4.svg" alt=""></a></li>
-                <li><a href=""><img src="images/portfolio3.svg" alt=""></a></li>
-                <li><a href=""><img src="images/portfolio5.svg" alt=""></a></li>                       
-            </ul>
-            <button class="btn-action" id="about-btn">Get my resume</button>
-          </div>
-          <div class="skill-container">
-            <div class="languages">
-                <h2>Languages</h2>
-                <i class="fa fa-chevron-down"></i>
+                    LET'S CONNECT
+                </a>
+                <ul class="social-media" id="social-media-desktop">
+                    <li><a href=""><img src="images/portfolio1.svg" alt=""></a></li>
+                    <li><a href=""><img src="images/Linkedin icon.svg" alt=""></a></li>
+                    <li><a href=""><img src="images/portfolio4.svg" alt=""></a></li>
+                    <li><a href=""><img src="images/portfolio3.svg" alt=""></a></li>
+                    <li><a href=""><img src="images/portfolio5.svg" alt=""></a></li>
+                </ul>
```

Comment:
Figma
![image](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/78251927/4dd942b3-13e8-4aa9-a7c5-14426802fca1)

Your design
![image](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/78251927/4889507c-3594-43ec-a8f2-cfb55e1afd49)

Please adjust the `font-size` of the elements in this section to match the design on Figma 🤓👍